stubs/seaborn/seaborn/palettes.pyi

```diff
@@ -0,0 +1,149 @@
+from collections.abc import Iterable, Sequence
+from typing import TypeVar, overload
+from typing_extensions import Literal, Self, TypeAlias
+
+from matplotlib.colors import Colormap, LinearSegmentedColormap, ListedColormap
+from matplotlib.typing import ColorType
+
+__all__ = [
+    "color_palette",
+    "hls_palette",
+    "husl_palette",
+    "mpl_palette",
+    "dark_palette",
+    "light_palette",
+    "diverging_palette",
+    "blend_palette",
+    "xkcd_palette",
+    "crayon_palette",
+    "cubehelix_palette",
+    "set_color_codes",
+]
+
+_T = TypeVar("_T")
+
+SEABORN_PALETTES: dict[str, list[str]]
+MPL_QUAL_PALS: dict[str, int]
+QUAL_PALETTE_SIZES: dict[str, int]
+QUAL_PALETTES: list[str]
+
+class _ColorPalette(list[_T]):
```

Comment:
Seems like the members get passed to `mpl.colors.rgb2hex`, which takes some sort of color objects, so I don't think this should be generic over an unbounded TypeVar.